### PR TITLE
Initial CJ Lobby and Build Hub UI Panel Locales (en, ja, ko)

### PR DESCRIPTION
**Pull request by PointerOffset**
This is the initial commit for locale strings that cover the Creator Jam Lobby world and Build Hub world UI panels. Languages include:

- American English ( @PointerOffset, @BaxterOttoman, @enziv )
- Japanese ( liltalia )
- Korean ( Holy_Water )

Please note the Japanese and Korean translations are partially machine translated from Google Translate. liltalia and Holy_Water did initial translations but new strings have been added or revised since this was last worked on by them. The Japanese and Korean locales should be revisited.

The `CJ.Lobby.ThemeReveal.Title` and `CJ.Lobby.ThemeReveal.Body` strings are no longer used. **They have not been removed from the `local.*.json` files yet as I have not yet validated doing so will cause problems with the current in-world configuration.** In the meantime, they can be ignored and left untranslated.

### CODE REVIEW

File: en/locale.en.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "CJ.BuildHub.Guidelines.Licensing.Subheading": "<i>Creative Commons CC-BY 4.0 license (Attribution)</i>",
+        "CJ.BuildHub.Guidelines.Precautions.Header": "<b>Precautions:</b>",
+        "CJ.BuildHub.Guidelines.Precautions.Body": "Use caution when importing assets. Assets with a large polygon count can cause performance issues.\r\n\r\n1-5000 polygon count is recommended.\r\n\r\n5000-50,000 polygon count is ok but use caution when using many assets with larger polygon counts.\r\n\r\nLarge file size may cause slow loading times for the user and possibly cause performance\r\nissues.",
+        "CJ.BuildHub.Guidelines.Misc.Header": "Miscellaneous:",
+        "CJ.BuildHub.Guidelines.Misc.Body": "Save your work in the inventory often.\r\n\r\nClean up any mess you make.\r\n\r\nAsk for permission when interacting with someone's unfinished work.\r\nDo not clean up other people’s areas unless specifically asked.\r\n<br><br>"
```
Comment (stiefeljackal) [inline en/locale.en.json]: This should be "messes" instead of "mess".

> Clean up any messes you make.

File: en/locale.en.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        "CJ.BuildHub.Guidelines.Building.Subheading": "\"I want to build!\"",
+        "CJ.BuildHub.Guidelines.Licensing.Header": "<b>Licensing:</b>",
+        "CJ.BuildHub.Guidelines.Licensing.Body": "Credit yourself. People want to know what you've made!\r\n\r\nUse <i>CC-0(Public Domain)</i> or <i>CC-BY(Attribution)</i> license assets.\r\n\r\nAll Creations in Creator Jam will be licensed at the end of the jam as:",
+        "CJ.BuildHub.Guidelines.Licensing.Subheading": "<i>Creative Commons CC-BY 4.0 license (Attribution)</i>",
+        "CJ.BuildHub.Guidelines.Precautions.Header": "<b>Precautions:</b>",
+        "CJ.BuildHub.Guidelines.Precautions.Body": "Use caution when importing assets. Assets with a large polygon count can cause performance issues.\r\n\r\n1-5000 polygon count is recommended.\r\n\r\n5000-50,000 polygon count is ok but use caution when using many assets with larger polygon counts.\r\n\r\nLarge file size may cause slow loading times for the user and possibly cause performance\r\nissues.",
```
Comment (stiefeljackal) [inline en/locale.en.json]: This should be "sizes" instead of "size".

> "Large file sizes may cause slow loading..."

**Review by stiefeljackal — COMMENTED** [review]
Just found tiny grammar issues. Otherwise, the the rest of the English locale looks good to me.

File: en/locale.en.json (hunk 1 above)
Comment (PointerOffset) [reply en/locale.en.json]: Corrected. Thank you!

File: en/locale.en.json (hunk 2 above)
Comment (PointerOffset) [reply en/locale.en.json]: Corrected. Thank you!

**Review by stiefeljackal — APPROVED** [review]

**Comment by stiefeljackal** [comment]
> Please note the Japanese and Korean translations are partially machine translated from Google Translate. liltalia and Holy_Water did initial translations but new strings have been added or revised since this was last worked on by them. The Japanese and Korean locales should be revisited.

@PointerOffset Thank you for noting this. Another PR can be created to update these partially machine translated strings. For now, I will approve the PR and merge it in. Thanks!

**Pull request merged**